### PR TITLE
New fixes

### PR DESCRIPTION
**Pull request by rachmatowicz**
Additional fixes to the distributable-ejb branch.

### CODE REVIEW

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/stateful/passivation/PassivationTestCaseSetup.java [hunk 1]
```diff
@@ -42,38 +43,33 @@
...
-        address.add("passivation-store", "infinispan");
-        address.protect();
-        return address;
-    }
+    /**
+     * This test setup originally depended upon manipulating the passivation-store for the default (passivating) cache.
+     * However, since WFLY-14953, passivation stores have been superceeded by bean-management-providers
+     * i.e. use /subsystem=distributable-ejb/infinispan-bean-management=default instead of /subsystem=ejb3/passicvation-store=infinispan
+     */
+    private static PathAddress INFINISPAN_BEAN_MANAGEMENT_PATH = PathAddress.pathAddress(PathElement.pathElement("subsystem", "distributable-ejb"),
```
Comment (pferraro) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/stateful/passivation/PassivationTestCaseSetup.java:51]: private static final

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/stateful/persistencecontext/StatefulUnitTestCase.java [hunk 2]
```diff
@@ -145,4 +108,39 @@ public void testNonExtended() throws Exception {
...
     }
+
+    /**
+     * This test originally depended upon manipulatng the passivation-store for the default (passivating) cache.
+     * However, since WFLY-14953, passivation stores have been superceeded by bean-management-providers
+     * i.e. use /subsystem=distributable-ejb/infinispan-bean-management=default instead of /subsystem=ejb3/passicvation-store=infinispan
+     */
+    static class StatefulUnitTestCaseSetup implements ServerSetupTask {
+
+        private static PathAddress INFINISPAN_BEAN_MANAGEMENT_PATH = PathAddress.pathAddress(PathElement.pathElement("subsystem", "distributable-ejb"),
```
Comment (pferraro) [inline testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/stateful/persistencecontext/StatefulUnitTestCase.java:119]: private static final

**Review by pferraro — CHANGES_REQUESTED** [review]

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/stateful/passivation/PassivationTestCaseSetup.java (hunk 1 above)
Comment (rachmatowicz) [reply testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/stateful/passivation/PassivationTestCaseSetup.java:51]: done

File: testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/stateful/persistencecontext/StatefulUnitTestCase.java (hunk 2 above)
Comment (rachmatowicz) [reply testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/stateful/persistencecontext/StatefulUnitTestCase.java:119]: done

**Comment by rachmatowicz** [comment]
I've pushed the changes.

**Pull request merged**